packages/components/select-v2/src/useSelect.ts

```diff
@@ -804,10 +827,14 @@ const useSelect = (props: ISelectV2Props, emit) => {
     initStates()
   })
   useResizeObserver(selectRef, handleResize)
-  useResizeObserver(selectionRef, resetSelectionWidth)
+  useResizeObserver(selectionRef, () => {
+    resetSelectionWidth()
+    updateTooltip()
+  })
```

Comment:
Do you think observe `wrapperRef` is better?

Reply:
yeah, it's better